db/db_impl/db_impl_secondary.cc

```diff
@@ -620,6 +620,44 @@ Status DBImplSecondary::TryCatchUpWithPrimary() {
   return s;
 }
 
+Status DBImplSecondary::GetLiveFilesStorageInfo(
+    const LiveFilesStorageInfoOptions& opts,
+    std::vector<LiveFileStorageInfo>* files) {
+  assert(files);
+  files->clear();
+  std::vector<LiveFileStorageInfo> results;
+  Status s;
+
+  ROCKS_LOG_INFO(immutable_db_options_.info_log,
+                 "DBImplSecondary::GetLiveFilesStorageInfo");
+  LogFlush(immutable_db_options_.info_log);
+
+  bool need_flush_memtable{true};
+  s = IsFlushMemtableNeededForGetLiveFiles(opts, need_flush_memtable);
```

Comment:
For secondary instance, `need_flush_memtable` should always be false. Why call a function to evaluate it?

Reply:
Will remove.